config/initializers/email_prefixer.rb

```diff
@@ -1,3 +1,3 @@
 EmailPrefixer.configure do |config|
-  config.stage_name = 'WHEEL'
+  config.stage_name = Rails.application.secrets.subject_prefix_for_outgoing_emails
```

Comment:
@mohitnatoo I see. So to start with `stage_name` should not be `WHEEL`. 

I will keep the behavior same and will change `config.application_name` to the application name. 

Whether to pass env name or not can be taken up in another issue.